coralproject/cay
# Added embed codes

## What does this PR do?

- Added embed code for iframe, non-iframe and standalone link

## How do I test this PR?


- Create or edit a form
- You should see the code for iframe and non-iframe embeds and the standalone form link

@coralproject/frontend
